### ISSUE TITLE
trr ready，tct 部署失败

### ISSUE THREAD

**Issue by cosven** [PingCAP-QE/Naglfar#123]
不确实是不是 tiup 的锅
```
Verbose debug logs has been written to /root/.tiup/logs/tiup-cluster-debug-2021-03-05-09-20-02.log.
  Warning  Install  54s  clustertopology-controller  deploy cluster failed(Process exited with status 1):
Error: init config failed: 10.0.2.42:9090: template: Prometheus:45: unexpected {{end}}
```

**Comment by mahjonp** [comment]
close by https://github.com/pingcap/tiup/pull/1185

**Comment by AstroProfundis** [comment]
tiup-cluster v1.3.3 has been yanked in the mirror, so if you install it now the latest available version will be v1.3.2. And we're working on a hotfix release v1.3.4.
